api/config/base/kustomization.yaml

```diff
@@ -25,8 +25,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: cloudfoundry/cf-k8s-api
-  newName: cloudfoundry/cf-k8s-api
-  newTag: latest
+  newName: cf-k8s-api
+  newTag: 1a1abf73-268e-4b7f-98c7-8074b6effe0e
```

Comment:
What's this?

Reply:
Fixed on commit https://github.com/cloudfoundry/cf-k8s-controllers/pull/248/commits/3a27a36419ea9c5882d6fa330e393fe9686362d6